.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: bash 
+sudo: true
+dist: trusty
+addons:
+  apt:
+    packages:
+      - dh-make
+      - devscripts
+      - debhelper
+      - git
+
+before_deploy:
+ - ./build-package.bash quanta-d51 "${TRAVIS_BRANCH}"
+ - ./build-package.bash quanta-t41 "${TRAVIS_BRANCH}"
+ - if [ ! -e deb ]; then mkdir deb && cp -a *.deb deb/; fi
+
+deploy:
+  - provider: bintray
+    file: .bintray-deb.json
+    user: $BINTRAY_USER
+    key: $BINTRAY_KEY
+    on:
+      branch: master
```

Comment:
Am I getting this right that it'll try to build every time, but only upload the packages on merge to master?